models/experimental/bloom_old/tt/bloom_attention.py

```diff
@@ -259,18 +259,18 @@ def forward(
 
         query_layer = query_layer.transpose(1, 2)
         query_layer = bloom_utils.torch2tt_tensor(query_layer, device)
-        reshaped_query_layer = ttm.tensor.reshape(query_layer, 1, batch_size * self.num_heads, q_length, self.head_dim)
+        reshaped_query_layer = ttnn.reshape(query_layer, 1, batch_size * self.num_heads, q_length, self.head_dim)
```

Comment:
use experimental reshape. or will have to wait till ttnn.reshape goes to main.
the arg syntax used here is not for ttnn.reshape